blockchain/flowkit.go

```diff
@@ -410,17 +410,14 @@ func (fk *flowKit) getAccountStorage(address flow.Address) (string, error) {
 	args := []string{fmt.Sprintf(`{"type":"Address","value":"0x%s"}`, address.Hex())}
 	val, _, err := fk.executeScript(StorageIteration, args)
 	if err != nil {
-		return "", err
+		return "{}", err
 	}
 
-	/* TODO: Parse account storage
-	_, err = ParseAccountStorage(val)
+	storage, err := jsoncdc.Encode(val)
 	if err != nil {
-		return "", err
+		return "{}", err
 	}
-	*/
-
-	return val.String(), nil
+	return string(storage), nil
```

Comment:
Maybe better to use https://pkg.go.dev/encoding/json#MarshalIndent

Reply:
The frontend already parses the returned json so there's no need really